AutomatedTesting/Gem/PythonTests/Prefab/CMakeLists.txt

```diff
@@ -19,14 +19,14 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED AND PAL_TRAIT_BUILD_HOST_TOOLS)
             AutomatedTesting.Assets
     )
 
-    ly_add_pytest(
-        NAME AutomatedTesting::PrefabTests_Periodic
-        TEST_SUITE periodic
-        TEST_SERIAL
-        PATH ${CMAKE_CURRENT_LIST_DIR}/TestSuite_Periodic.py
-        RUNTIME_DEPENDENCIES
-            Legacy::Editor
-            AZ::AssetProcessor
-            AutomatedTesting.Assets
-    )
+    #ly_add_pytest(
```

Comment:
Any reason for commenting this out? Does running a suite with 0 tests cause any issues?

Reply:
Yes, it will actually "fail" the suite if it runs empty. I could remove this, but I believe we have at least one new test in the works that will shortly be added to TestSuite_Periodic and this makes for an easier re-enable of the Periodic test suite.

Reply:
Sounds like a bug if it marks it as a failure when running an empty suite. I suggest that you create a ghi for build folks to look into.

Reply:
I don't think it's a build issue. Pytest returns an error if there are no tests found on purpose so you're alerted if mistakenly running an empty suite.

Reply:
Raising when zero tests are run helps avoid accidentally disabling all tests and getting a "green" signal from running no tests.